var/spack/repos/builtin/packages/py-strawberryfields/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PyStrawberryfields(PythonPackage):
+    """Open source library for continuous-variable quantum computation"""
+
+    homepage = "https://github.com/XanaduAI/StrawberryFields"
+    pypi = "StrawberryFields/StrawberryFields-0.23.0.tar.gz"
+
+    maintainers = ["marcodelapierre"]
```

Comment:
This will override the Python-package maintainers (e.g., @adamjstewart).  Not sure how he feels about it but you could do the following so they are notified as well:

```suggestion
    maintainers = PythonPackage.maintainers + ["marcodelapierre"]
```